src/seedu/addressbook/commands/Command.java

```diff
@@ -46,6 +46,14 @@ public CommandResult execute(){
     //Note: it is better to make the execute() method abstract, by replacing the above method with the line below:
     //public abstract CommandResult execute();
 
+    /**
+     * Describes whether a command mutates the AddressBook. Assumed to be false by default to prevent functions from
```

Comment:
Perhaps it would be more precise to just mention that isMutating() returns true if the command can possibly change the AddressBook's data.